src/recoil/index.ts

```diff
@@ -4,6 +4,26 @@ import { DagStateNodesLinksStrat, OntologyVertexDatum } from "../types/graph";
 import { DatasetGraph, Ontology } from "../types/d";
 import { createDatasetGraph, createLattice } from "../util/loadDatasetGraph";
 
+export const isProdState = selector<boolean>({
+  key: "isProd",
+  get: () => {
+    return import.meta.env.MODE === "production"; // this comes from vite
+  },
+});
+
+export const apiPrefixState = selector<string>({
```

Comment:
Is this worth making in Recoil or perhaps it's better off being a variable in `globals.ts` that you could import? Having to get the api prefix state in all the other selectors adds a lot of verbosity. It's also not important to make this in Recoil because we don't expect the api prefix state to be changing dynamically during the operation of the app.

Reply:
Moved to util

Reply:
https://github.com/chanzuckerberg/ontology-ui/pull/154/commits/0d3b2a89096c06adff977b9842f49d65191ebf6d